frontend/src/component/admin/apiToken/ProjectsList/ProjectsList.tsx

```diff
@@ -20,75 +20,68 @@ interface IProjectsListProps {
+                </HtmlTooltip>
             </TextCell>
         );
     }
 
-    const fieldsHead = fields.length < 5 ? fields : fields.slice(0, 3);
-    const fieldsTail = fields.length < 5 ? [] : fields.slice(3);
-
-    return (
-        <TextCell>
-            {fieldsHead.map((item, index) => (
-                <Fragment key={item}>
-                    {index > 0 && ', '}
-                    {!item || item === '*' ? (
+    if (
+        (projectsList.length === 1 && projectsList[0] === '*') ||
+        project === '*' ||
+        (!project && (!projectsList || projectsList.length === 0))
+    ) {
+        return (
+            <TextCell>
+                <HtmlTooltip
+                    title='ALL current and future projects'
+                    placement='bottom'
+                    arrow
+                >
+                    <span>
                         <Highlighter search={searchQuery}>*</Highlighter>
-                    ) : (
-                        <StyledLink to={`/projects/${item}`}>
-                            <Highlighter search={searchQuery}>
-                                {item}
-                            </Highlighter>
-                        </StyledLink>
-                    )}
-                </Fragment>
-            ))}
-            <ConditionallyRender
-                condition={fieldsTail.length > 0}
-                show={
-                    <>
-                        {', '}
-                        <HtmlTooltip
-                            title={fieldsTail.map((item, index) => (
-                                <Fragment key={item}>
-                                    {index > 0 && ', '}
-                                    {!item || item === '*' ? (
-                                        <Highlighter search={searchQuery}>
-                                            *
-                                        </Highlighter>
-                                    ) : (
-                                        <StyledLink to={`/projects/${item}`}>
-                                            <Highlighter search={searchQuery}>
-                                                {item}
-                                            </Highlighter>
-                                        </StyledLink>
-                                    )}
-                                </Fragment>
-                            ))}
-                            placement='bottom-start'
-                            arrow
-                        >
-                            <span>+{`${fieldsTail.length}`} more</span>
-                        </HtmlTooltip>
-                    </>
-                }
-            />
-        </TextCell>
-    );
+                    </span>
+                </HtmlTooltip>
+            </TextCell>
+        );
+    }
+
+    if (projectsList.length === 1 || project) {
+        const item = project || projectsList[0];
+
+        return <LinkCell to={`/projects/${item}`} title={item} />;
```

Comment:
Does this still render the project's ID if there is only one ID? If so, won't `wwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww` still cause problems with the height of the table row?

Reply:
Link cell already has truncation built-in

Reply:
Ah, great! Thanks.